### PR TITLE
feat(dev-console): Refactored Home Service as Apps service

### PR DESCRIPTION
**Pull request by deshmukhmayur**
# Resolves 947

# Explain the feature/fix

Will act as the backend for the Developer Console.

## Does this PR introduce a breaking change

Yes.

Renames the home-service as apps service, so some of the integrations with feedback and other microservices will need some changes.

## Screenshots

N/A

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [ ] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] Was this feature demo'd and the design review approved?

### CODE REVIEW

File: packages/api-gateway-service/config.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "subscriptionsUri": "ws://feedback-service:8080/subscriptions"
   },
   {
-    "name": "Home",
+    "name": "Apps",
     "uri": "http://home-service:8080/graphql",
```
Comment (riginoommen) [inline packages/api-gateway-service/config.json:14]:
```suggestion
    "uri": "http://apps-service:8080/graphql",
```

File: packages/api-gateway-service/config.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
...
     "subscriptionsUri": "ws://home-service:8080/subscriptions"
```
Comment (riginoommen) [inline packages/api-gateway-service/config.json:15]:
```suggestion
    "subscriptionsUri": "ws://apps-service:8080/subscriptions"
```

**Review by riginoommen — CHANGES_REQUESTED** [review]

File: packages/api-gateway-service/config.json (hunk 1 above)
Comment (deshmukhmayur) [reply packages/api-gateway-service/config.json:14]: Good catch. I forgot to change that.

Comment (deshmukhmayur) [reply packages/api-gateway-service/config.json]: Updated the commit.

File: packages/api-gateway-service/config.json (hunk 2 above)
Comment (deshmukhmayur) [reply packages/api-gateway-service/config.json]: Updated the commit.

**Review by riginoommen — APPROVED** [review]

**Review by ghanlohar — APPROVED** [review]

**Pull request merged**